src/main/java/io/bf2/kafka/topic/ManagedKafkaCreateTopicPolicy.java

```diff
@@ -56,7 +54,6 @@ public void validate(RequestMetadata requestMetadata) throws PolicyViolationExce
         }
 
         validateReplicationFactor(requestMetadata);
-        validateIsr(requestMetadata);
```

Comment:
Why are we removing ISR validation? As I understand the card/PR its just about removing a specific set of default config values but we would still expect there to be a `defaultIsr` and `defaultReplicationFactor` wouldn't we?

Reply:
Something to clarify:
`validateIsr` method: validate `min.insync.replica` config
`validateReplicationFactor` method: validate replicationFactor, which is not part of topic config.

So, after topic config policy applied, we validate `min.insync.replica` config in enforced rule, to only allow 2 for standard instance, and 1 for developer instance. Thus, we don't need validate it separately(via validateIsr` method) anymore. 

For replication factor, it's not part of topic config, so it will be kept.
Hope it's clear. Thanks.

Reply:
Ok, I think I understand why this PR deletes `validateIsr` though I'm now confused as to why `validateIsr` was implemented as it was...